crates/asset-importer-rs-scene/src/material.rs

```diff
@@ -474,10 +538,10 @@ impl AiMaterial {
         semantic_type: Option<AiTextureType>,
         index: u32,
     ) -> Option<Result<String, FromUtf8Error>> {
-        self.get_property_type_info(key, semantic_type, index)
-            .and_then(|x| match x {
-                AiPropertyTypeInfo::Binary(binary) => {
-                    let str = String::from_utf8(binary.to_vec());
+        self.get_property(key, semantic_type, index)
+            .and_then(|prop| match prop.property_type {
+                AiPropertyTypeInfo::Binary => {
+                    let str = String::from_utf8(prop.data.to_vec());
```

Comment:
Compliment: `x` was me being lazy with lambda expressions. renaming to `prop` is proper.